tox.ini

```diff
@@ -4,7 +4,7 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py27, py34, py35, py36, py37, py38, pypy3
+envlist = py36, py37, py38, py39, pypy3
```

Comment:
Does this mean that we don't run tox in python 2.7?

Reply:
My bad, I will return it . Good catch.